test/unit/sql/where.test.js

```diff
@@ -908,6 +908,78 @@ suite(Support.getTestDialectTeaser('SQL'), () => {
       });
     }
 
+    if (current.dialect.supports.REGEXP) {
+      suite('$regexp', () => {
+        testsql('username', {
+          $regexp: '^sw.*r$'
+        }, {
+          mysql: "`username` REGEXP '^sw.*r$'",
+          postgres: '"username" ~ \'^sw.*r$\''
+        });
+      });
+
+      suite('$regexp', () => {
```

Comment:
I don't understand why this suite is here twice, if the first test worked I don't see a possibility the second one wouldn't